policyengine_us/variables/gov/hhs/chip/is_chip_eligible_standard_pregnant_person.py

```diff
@@ -0,0 +1,49 @@
+from policyengine_us.model_api import *
+
+
+class is_chip_eligible_standard_pregnant_person(Variable):
+    value_type = bool
+    entity = Person
+    label = "Pregnant person eligible for standard CHIP"
+    documentation = "Determines if a pregnant person is eligible for the standard Children's Health Insurance Program"
+    definition_period = YEAR
+    reference = (
+        "https://www.ssa.gov/OP_Home/ssact/title21/2110.htm",
+        "https://www.medicaid.gov/medicaid/national-medicaid-chip-program-information/medicaid-childrens-health-insurance-program-basic-health-program-eligibility-levels",
+    )
+
+    def formula(person, period, parameters):
+        # Get state code
+        state_code = person.household("state_code", period)
+
+        # Check pregnancy status
+        is_pregnant = person("is_pregnant", period)
+
+        # Check if state offers CHIP program for pregnant women
+        p = parameters(period).gov.hhs.chip.pregnant
+        income_limit = p.income_limit[state_code]
+
+        state_has_pregnant_chip = income_limit > 0
+
+        # Check immigration status eligibility
+        istatus = person("immigration_status", period)
+        undocumented = istatus == istatus.possible_values.UNDOCUMENTED
```

Comment:
add source for this requirement - neither of those in the variable corroborate it